emcc

```diff
@@ -708,7 +708,7 @@ try:
   if js_opts is None: js_opts = opt_level >= 2
   if llvm_opts is None: llvm_opts = LLVM_OPT_LEVEL[opt_level]
   if opt_level == 0: debug_level = max(3, debug_level)
-  if memory_init_file is None: memory_init_file = opt_level >= 2
+  if memory_init_file is None: memory_init_file = 2 if opt_level >= 2 else 0
```

Comment:
It is possible to do the requests in parallel. But I didn't really consider until now that if this is efficient enough, it could be better than even that. I wonder how to measure it though.

Regardless, until we have such measurements, I don't think we should change the default - we shouldn't confuse users with changes unless they have proven benefits.